quicklogic/pp3/utils/create_place_constraints.py

```diff
@@ -102,39 +128,31 @@ def main():
         # Search for a CLOCK cell connected to that net
         for cell in eblif_data["subckt"]:
             if cell["type"] == "subckt" and cell["args"][0] == IOB_CELL[0]:
```

Comment:
I think it is better to define a variable rather than calling IOB_CELL index. I guess index 0, 1 and 2 has specific meaning.

Reply:
Alright. I've changed the tuple to a dict so that it is more readable. I have also added a test design which utilizes all global clocks.